plugins/protocolbuffers/pyi/v26.1/buf.plugin.yaml

```diff
@@ -14,5 +14,5 @@ registry:
     requires_python: ">=3.8"
     deps:
       # https://pypi.org/project/protobuf/
-      - "protobuf~=5.26"
-      - "types-protobuf"
+      - "protobuf~=5.26.1"
+      - "types-protobuf~=5.26"
```

Comment:
guess we could have done this in the v27 plugins now that there's a release for them, but probably not worth another rev this time.

Reply:
I paused the release workflow as they haven't gone out yet but I can update v27.1 at least with this.

Reply:
https://github.com/bufbuild/plugins/pull/1322